public/v4/apps/chadburn.yml

```diff
@@ -0,0 +1,33 @@
+captainVersion: 4
+version: 3.3
+services:
+    '$$cap_appname':
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock:ro
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM premoweb/chadburn:$$cap_version
+                - CMD ["daemon"]
```

Comment:
Looks like the default CMD is `daemon`. 
https://github.com/PremoWeb/chadburn/blob/9db334edebdb034c26c5f0c0ead57226930c37a3/Dockerfile#L18

If that's correct, then we don't need to override it here. Instead of `dockerfileLines`, you can simple add an `image` field (sibling to `volume` a few lines before this). See here for an example:
https://github.com/caprover/one-click-apps/blob/0ad28e01b57eccb78d5b53ea2901844ec4736c6c/public/v4/apps/mysql.yml#L3-L6

Reply:
Updated! Thank you.